frontend/src/assets/about/imprint.html

```diff
@@ -0,0 +1 @@
+<p>Hallo</p>
```

Comment:
Hallo!

(Shouldn't this be exchanged?)

Reply:
This file shouldn't have ever been committed :sweat_smile:. I've removed in https://github.com/hpi-sam/digital-fuesim-manv/pull/643/commits/450a96f6150c6c6c206b56190c15520caf15f1c6